src/shared/components/oncoprint/SortUtils.ts

```diff
@@ -302,6 +303,32 @@ export function getClinicalTrackSortComparator(track: ClinicalTrackSpec) {
     };
 }
 
+export function getClinicalTrackInitialSortDirection(track: ClinicalTrackSpec) {
+    let initialSortOrder: 0 | 1 | -1 | undefined = 0 as 0;
+    getServerConfig().oncoprint_clinical_tracks_show_by_default.forEach(t => {
+        if (t.stableId === track.attributeId) {
+            if (t.sortOrder === 'ASC') {
+                initialSortOrder = 1;
+            } else if (t.sortOrder === 'DESC') {
+                initialSortOrder = -1;
+            }
+        }
+    });
+    return initialSortOrder;
+}
+
+export function getClinicalTrackInitialShowGapsConfig(
+    track: ClinicalTrackSpec
+) {
+    let gapOn: boolean = false;
+    getServerConfig().oncoprint_clinical_tracks_show_by_default.forEach(t => {
+        if (t.stableId === track.attributeId) {
+            gapOn = t.gapOn === true ? (gapOn = true) : (gapOn = false);
+        }
+    });
+    return gapOn;
+}
```

Comment:
Simplify to:
```
export function getClinicalTrackInitialShowGapsConfig(
    track: ClinicalTrackSpec
): boolean {
    getServerConfig().oncoprint_clinical_tracks_show_by_default.forEach(t => {
        if (t.stableId === track.attributeId) {
            return !!t.gapOn;
        }
    });
    return false;
}
```

Reply:
Looks like returning inside forEach is not working, this is returning all the time false. I updated the code and used he !!t.gapOn to simplify.